tests/cluster/check_vagrant.py

```diff
@@ -39,12 +41,13 @@ class CheckVagrantCluster(object):
 
 """
 
+    cluster_file = "cluster_file_temporary.json"
+
     def check_one_host_parsing(self, monkeypatch):
         self._vagrant_ssh_data = self.two_hosts
         monkeypatch.setattr("ducktape.cluster.vagrant.VagrantCluster._vagrant_ssh_config", lambda vc: (self._vagrant_ssh_data, None))
-        monkeypatch.setattr("ducktape.cluster.vagrant.VagrantCluster._is_aws", lambda vc: False)
-        monkeypatch.setattr("ducktape.cluster.vagrant.VagrantCluster._externally_routable_ip", lambda vc, is_aws, node_account: "127.0.0.1")
-
+        monkeypatch.setattr("ducktape.cluster.vagrant.VagrantCluster.is_aws", lambda vc: False)
```

Comment:
Looks like monkeypatching behavior is identical in these 3 tests now - maybe break this out into a shared method?